server/handlers/stakeholderHandler.js

```diff
@@ -33,214 +16,230 @@ const stakeholderGetQuerySchema = Joi.object({
   email: Joi.string(),
   phone: Joi.string(),
   website: Joi.string(),
-  children: Joi.array(),
-  parents: Joi.array(),
-  users: Joi.array(),
-  filter: Joi.object(),
+  search: Joi.string(),
+}).unknown(false);
+
+const stakeholderPostSchema = Joi.object({
+  type: Joi.string(),
+  email: Joi.string(),
+  phone: Joi.string(),
+}).unknown(false);
+
+const stakeholderDeleteSchema = Joi.object({
+  id: Joi.string().uuid(),
+  type: Joi.string(),
+  linked: Joi.boolean(),
+  data: Joi.object({
+    id: Joi.string().uuid().required(),
+    email: Joi.string().email(),
+    first_name: Joi.string(),
+    last_name: Joi.string(),
+    org_name: Joi.string(),
+    logo_url: Joi.string(),
+    map: Joi.string(),
+    phone: Joi.string(),
+    website: Joi.string().uri(),
+    children: Joi.array().items(Joi.object()),
+    parents: Joi.array().items(Joi.object()),
+    image_url: Joi.string().allow(''),
+    type: Joi.string(),
+    created_at: Joi.string(),
+    updated_at: Joi.string(),
+  })
+    .unknown(false)
+    .xor('org_name', 'first_name')
+    .xor('org_name', 'last_name'),
```

Comment:
@gwynndp I added these extra validations that basically ensure that either first_name and last_name are present or org_name is present but not both. Is that fine?

Reply:
Love the logic, but I think it may need to be applied to only the data property object because it's throwing errors.  I tried simply moving it and couldn't get it to work.  I slacked you a sample object for testing.